app/src/main/java/org/mozilla/focus/telemetry/GleanMetricsService.kt

```diff
@@ -79,7 +80,9 @@ class GleanMetricsService(context: Context) : MetricsService {
     private fun collectPrefMetrics(
         components: Components,
         settings: Settings
-    ) = CoroutineScope(IO).async {
+    ) = CoroutineScope(Main).async {
```

Comment:
Otherwise I'd get a `java.lang.IllegalThreadStateException: Must have a Handler` the same as was seen in https://github.com/mozilla-mobile/fenix/issues/17045